README.md

```diff
@@ -3,3 +3,5 @@
 Contributors
 
 1. Aditya Agarwal
+
+Yash
```

Comment:
Do these things->

1. Get the latest code of my master branch to your `yash` branch. The command for that would be

```sh
git checkout yash # switch to yash branch
git pull upstream master # get the code of upstream's master branch in your local's yash branch.
```

If the second command gives an error then most likely you have not added the upstream remote to your local git repo. To do that use this command

```sh
git remote add upstream https://github.com/itaditya/road-to-hacktober.git
```